src/frontends/src/query_frontend_main.cpp

```diff
@@ -12,14 +12,19 @@ int main(int argc, char* argv[]) {
                            "Clipper query processing frontend");
   // clang-format off
   options.add_options()
-    ("redis_ip", "Redis address", cxxopts::value<std::string>()->default_value("localhost"))
-    ("redis_port", "Redis port", cxxopts::value<int>()->default_value("6379"));
+    ("redis_ip", "Redis address",
+        cxxopts::value<std::string>()->default_value("localhost"))
+    ("redis_port", "Redis port",
+        cxxopts::value<int>()->default_value("6379"))
+    ("threadpool_size", "Number of threads for the task execution threadpool",
+        cxxopts::value<int>()->default_value("4"));
```

Comment:
Agreed. I filed [CLIPPER-101](https://clipper.atlassian.net/browse/CLIPPER-101) to explore this. I don't think we can make this decision until the system is completed though.